nix/overlay.nix

```diff
@@ -0,0 +1,28 @@
+# Override some packages and utilities in 'pkgs'
+# and make them available globally via callPackage.
+#
+# For more details see:
+# - https://nixos.wiki/wiki/Overlays
+# - https://nixos.org/nixos/nix-pills/callpackage-design-pattern.html
+
+self: super:
+
+let
+  /* FIXME not sure if the stdenv and other are usefull*/
+  inherit (super) stdenv stdenvNoCC callPackage;
```

Comment:
Since none of those are used in this file then inheriting them from `super` is pointless.

Also, we might want to change the arguments for the overlay from `self` and `super` to ones that actually make some sense, for example `final` and `orev`, like the [wiki article describes](https://nixos.wiki/wiki/Overlays). That way it's more clear that:

* The `final` argument represents the state of nixpkgs after overlay is applied.
* The `prev` argument represents the state of nixpkgs before overlay is applied.